frontend/public/kubevirt/components/utils/lazyRenderer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import * as _ from 'lodash-es';
+import { inject } from '../../../components/utils';
+
+const DEFAULT_THROTTLE_DELAY = 500; // in ms
+
+/**
+ * Avoids eager re-rendering for frequent redux store changes.
+ *
+ * Assumption: re-render requests are frequent, so skipping a few of them
+ * has no impact on rendered information from user's perspective.
+ *
+ * If that assumption is not met, the use of this component is not recommended.
+ * Or the component needs to be modified to ensure the render() phase is reliably called, ie.
+ * by updating inner state to ensure the shouldComponentUpdate() function is called.
+ */
```

Comment:
I would rather implement this here; to not pass any assumptions on a consumer. So that render is reliably called.

Reply:
Ok, I tried :-)
Such an improvement should not be complicated.

Reply:
Actually, it even simplified the code.

Reply:
looks nice :)